### PR TITLE
Minor typo and grammar fixes

### PR DESCRIPTION
**Pull request by kdpullen95**
#### Summary
Content "Minor typo and grammar fixes for various pieces of content."

#### Purpose of change
Just fixing grammar issues and typos that I've personally encountered during recent playthroughs (and any similar issues like dont->don't that appeared in VSCode when I searched up the typo). I tried to stick as close to the original wording as possible, but there's a few areas where I took what I believe are mild liberties. I didn't touch typos in notes or anywhere else where typos would make sense.

#### Describe the solution
Just some grammar issues and typo fixes.

#### Describe alternatives you've considered
I considered combing through and doing a proper proofread but just editing stuff I come across as I play is more practical and fun.

#### Testing
String edits only.

#### Additional context
The string concatenation used for hallucinated sounds already handles adding spaces, so that's why the spaces at the end of the vertical directions were removed. It was resulting in double spaces in the middle of a sentence.

This is my first time opening a PR into a repo from a forked one. I think I followed the instructions correctly, but please let me know if I did something incorrectly.

### CODE REVIEW

**Comment by kdpullen95** [comment]
Unsure why the build is failing for changes as benign as this, I'll look into it once the weekend rolls around again.

**Comment by sadenar** [comment]
Hi, first, really appreciate somebody hunting typos, second, look at those I collected and decide whether you want to fix them too.

Heat Tolerance mutation's description could use a tone change to it making heatstroke less likely to occur, it's pretty evident that getting burnt is gonna hurt no matter if you're heat resistant.

The Zoroastrian holy symbol necklace's description, the Faravahar, has a typo where it should be believed instead of believer:
- He stands above a pair of wings that are outstretched from a circle which are believer to represent eternity. -> He stands above a pair of wings that are outstretched from a circle which are believed to represent eternity.

Double period at the end of this Isherwood dialogue string:
- Things have been tough for my dad since Barry was captured.  He works till he falls asleep most nights..

As for XE, the new planar/magic attuned content has a few inconsistant uses of capitalization:
- Water Sprites -> water sprites
- Smoke Mephits -> smoke mephits
- ratatosk's child -> Ratatosk's child
- nimue's dragonfly - Nimue's dragonfly

That's all I've found so far, usually I just wrap it up in a neat little issue somebody can just fix whenever they want, but since you made this PR, I figured this could be added to it.

**Comment by sadenar** [comment]
One more just recently added, bad capitalization at the start:
-Alloying reference manual -> alloying reference manual

**Comment by kdpullen95** [comment]
Couldn't replicate the previous test failure on my machine (all green), and I'm really not sure how my changes could've caused that one in particular to fail, anyway.  Is it possible it's test flake? Not sure how common that is in this repo.

Added some more typo fixes in the interim. Hoping the tests pass now, and will add the rest of @sadenar 's suggestions after work (thank you, by the way!).

**Comment by sadenar** [comment]
Dunno if you're still working on this but I went ahead and fixed everything I found in an issue of my own, thanks for expressing interest though.

**Comment by kdpullen95** [comment]
Glad it's getting fixed through your efforts! I'm still adding to mine behind the scenes as I find more typos, but I was stuck the past couple weeks trying to figure out the test failure from the first build run.

I just realized from checking back due to your comment that someone relaunched the GitHub Actions and now it's failing on an incorrect typo check - I guess I can revert that and ask for someone else to relaunch the Actions again for a third check. I'm not too happy about the fact that a typo needs to exist or the build fails, though.

**Comment by kdpullen95** [comment]
Alright, finally was able to fix the one failure and get it to run all passing on my machine. Here's proof:
image

Going to put this into ready for review now, if anyone's can approve the workflow runs.

**Review by LyleSY — APPROVED** [review]
DinoMod bits look fine

**Comment by LyleSY** [comment]
Kicked

File: data/mods/Xedra_Evolved/monsters/changeling.json [hunk 1]
```diff
@@ -382,7 +382,7 @@
     "id": "mon_goblin_spider",
     "type": "MONSTER",
     "name": { "str": "goblin spider" },
-    "description": "A giant, dark gray goblin spider, the size of a pony and above the mandibles where you would expect to see the eyes instead there is a child's face.  It seems to vanish into the shadows if you take your eyes off it.",
+    "description": "A giant, dark gray goblin spider, the size of a pony. Above the mandibles, where you would expect to see the eyes, instead there is a child's face.  It seems to vanish into the shadows if you take your eyes off it.",
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/monsters/changeling.json:385]:
```suggestion
    "description": "A giant, dark gray goblin spider, the size of a pony.  Above the mandibles, where you would expect to see the eyes, instead there is a child's face.  It seems to vanish into the shadows if you take your eyes off it.",
```

**Pull request merged**